install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -x
+set -e
+
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root."
+   exit 1
+fi
+
+PREFIX=/usr
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+# Install JobAdder source code + binaries
+# FIXME: how to make setup.py ignore test folder??
+cd $SCRIPTPATH
+mv ./src/test/__init__.py ./src/test/init
```

Comment:
Does this prevent the installation of test?

Reply:
Yes, without `__init__.py` is is not detected as python module and not installed.